### ISSUE TITLE
Easy way to get usage / quota information

### ISSUE THREAD

**Issue by genixpro** [Factual/factual-nodejs-driver#10]
I know that the Factual API provides ones current quota/usage information in the form of the "X-Factual-Throttle-Allocation" header. It would be nice if this library made that information available in an easy to use form.

**Comment by easherma** [comment]
I agree! If you enable debug, this info gets printed. I wouldn't think it would be much of a stretch to store that data as an object instead?
